src/core/dbus-cgroup.c

```diff
@@ -345,6 +346,22 @@ static int property_get_ip_address_access(
         return sd_bus_message_close_container(reply);
 }
 
+/* Stub getter since BPFProgram= property does not store data in CGroupContext. */
```

Comment:
it should though...

i.e. the path to the bpffs progs should be store in CGroupContext. The runtime references to them (i.e. the fds) in Unit itself.

Reply:
Updated with storing path in `CGroupContext` and fds in `Unit`.

Also implemented a getter:
```
[root@image src]# systemctl show bpf-foreign | grep BPFProgram
BPFProgram=egress:/sys/fs/bpf/trivial/cgroup_skb_egress
```